### PR TITLE
Add reveal support

### PR DESCRIPTION
**Pull request by willm30**
This PR adds support for the Reveal widget. Test with https://github.com/JoinColony/colonyCDapp/pull/385

### CODE REVIEW

**Review by rdig — APPROVED** [review]
Yup, seems to work as good

File: src/handlers/motions/motionVoteRevealed/index.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  const votingClient = await getVotingClient(colonyAddress);
+  const { chainId } = await votingClient.provider.getNetwork();
+  const motionDatabaseId = getMotionDatabaseId(
+    chainId,
+    votingClient.address,
+    motionId,
+  );
+  const revealedMotion = await getMotionFromDB(colonyAddress, motionDatabaseId);
+
+  if (revealedMotion) {
```
Comment (jakubcolony) [inline src/handlers/motions/motionVoteRevealed/index.ts:24]: As in the other one, should we log if not found?

**Review by jakubcolony — APPROVED** [review]
All good here! 👍

File: src/handlers/motions/motionVoteRevealed/index.ts (hunk 1 above)
Comment (willm30) [reply src/handlers/motions/motionVoteRevealed/index.ts:24]: Gets logged inside getMotionFromDB

**Pull request merged**